release-protocol.md

```diff
@@ -0,0 +1,37 @@
+# Release protocol
+
+## Alert the team
+
+A release PR should not happen while there are remaining work items in code reivew or QA.
+
+Consult the team if there are still items in QA, or if other previous merges need to be omitted from the release.
+
+The Development branch should be locked during the release procedure.
+Note: this locked state should already be the case while the Development branch is being QAed (refer to the pull request template).
+
+## Merge and create the PR
+
+First, resolve any potential merge conflicts with main on the develop branch.
+
+1. `git checkout master`
+2. `git pull origin master`
+3. `git checkout Development`
+4. `git pull origin Development`
+5. `git merge master` (there should be no merge conflicts)
+6. Update the version in symbbase.php.
+7. Add the version change, commit, and push to Development.
+
+Then, issue a pull request for merging the Development branch into the master branch. Await approval, and then merge. Note: do NOT use the "squash and merge" method; it will make subsequent merges with the Development branch more difficult. Instead, use the "Create a merge commit" option.
+
+Note: if you encounter merge conflicts when merging Development into master, something went wrong. Investigate what happened thoroughly before continuing.
+
+## Draft a new release
+
+After completion of the previous steps, draft a new release of the master branch on GitHub:
+
+1. Navigate to [https://github.com/BioKIC/Symbiota/releases/new](https://github.com/BioKIC/Symbiota/releases/new).
+2. Designate the target as the master branch.
+3. Create a new tag following the SemVer pattern outlined [here](https://semver.org/) (vX.Y.Z). Note that this exact pattern (no period between v and the first name, for instance) must be followed explicitly. The release title should be the same as the tag. You can leave the release description blank.
```

Comment:
@arbolitoloco I'm open to the release title not mirroring the tag, but I checked a few other open source projects out there, and it's hard to tell whether I just can't see the release title or if it's just that it matches the tag (i.e., both are just the SemVer version).

I'm fine either way, but I have a minor preference to follow whatever the convention is. Thoughts?

Reply:
@Atticus29 Since there are no conventions for the "title" field in the GitHub releases, I'd only advocate for adding the name of the software in front of the version if we're adding anything. For explanations, please check [this Stack Exchange conversation](https://softwareengineering.stackexchange.com/a/440017).

Reply:
> "This release title is what GitHub will also assign to the website release name but also to the zipped download file name as well. That is why it is good practice to give your release a full text name with a version number at the end."

Ok. You've convinced me that this is a good idea. Let's do it!